ledger/src/blockstore.rs

```diff
@@ -3288,6 +3288,23 @@ fn send_signals(
     }
 }
 
+/**
+ * For each slot in the slot_meta_working_set which has any change, update
+ * its meta-data entry in the cf::SlotMeta.
```

Comment:
nit: can you use `metadata` instead of `meta-data`; looks like we use `metadata` in a bunch of other comments